README.md

```diff
@@ -57,13 +57,17 @@ Commands:
 You can use caching by including this module in your ActiveRecord models,
 
     class OfferEvent < ActiveRecord::Base
-      include Gris::CacheKey
+      include Gris::Caching
     end
 
 and then cache inside GET requests like this
 
     offer_event =  OfferEvent.cached_find(id)
 
+To expire the cache for an object, call
+
+    OfferEvent.expire_cache(id)
```

Comment:
@bhoggard what do you think about an instance method that wraps some of this such that we can `offer_event.expire_cache` 

Otherwise, could we update the text here so that its clear that we are expiring the cache for a given key rather than for an object.